frontend/src/components/pages/Dashboard.tsx

```diff
@@ -40,7 +39,6 @@ const Default = (): React.ReactElement => {
       <Wrap>
         <Logout />
         <RefreshCredentials />
-        <ResetPassword />
```

Comment:
quick note to reviewer(s) - since `<ResetPassword />` used to just be a Button on the dashboard but is now a full component, it gets rendered weirdly in the corner if left on the dashboard page, so I took it out

Reply:
✅